gradle/libs.versions.toml

```diff
@@ -0,0 +1,31 @@
+[versions]
+agp = "8.2.0-beta06"
+kotlin = "1.9.20"
+dokka = "1.9.10"
+ksoup = "0.0.6"
+vanniktect-publish = "0.25.3"
+ktor = "2.3.6"
+
+
+compileSdk = "34"
+minSdk = "24"
+libraryVersion = "1.8.0"
+
+
+[libraries]
+kotlin-test = { module = "org.jetbrains.kotlin:kotlin-test", version.ref = "kotlin" }
+ksoup = { module = "com.fleeksoft.ksoup:ksoup", version.ref = "ksoup" }
+ktor-client-core = { module = "io.ktor:ktor-client-core", version.ref = "ktor" }
+ktor-client-logging = { module = "io.ktor:ktor-client-logging", version.ref = "ktor" }
+ktor-client-darwin = { module = "io.ktor:ktor-client-darwin", version.ref = "ktor" }
+ktor-client-mock = { module = "io.ktor:ktor-client-mock", version.ref = "ktor" }
+ktor-client-okhttp = { module = "io.ktor:ktor-client-okhttp", version.ref = "ktor" }
+
+[plugins]
+androidApplication = { id = "com.android.application", version.ref = "agp" }
+androidLibrary = { id = "com.android.library", version.ref = "agp" }
+kotlinAndroid = { id = "org.jetbrains.kotlin.android", version.ref = "kotlin" }
+kotlinMultiplatform = { id = "org.jetbrains.kotlin.multiplatform", version.ref = "kotlin" }
+kotlinCocoapods = { id = "org.jetbrains.kotlin.native.cocoapods", version.ref = "kotlin" }
+dokka = { id = "org.jetbrains.dokka", version.ref = "dokka" }
+vanniktectPublish = { id = "com.vanniktech.maven.publish.base", version.ref = "vanniktect-publish" }
```

Comment:
nit: there's no extra line break at the end of this file. you might want to update your IDE settings.

Reply:
Will remove it. Thanks